packages/navigator/src/components/NavigatorTable.vue

```diff
@@ -69,6 +69,12 @@ import { mapState } from 'vuex'
 export default {
   name: 'NavigatorTable',
   data () {
+    let dataExplorerUrl = window.__INITIAL_STATE__.pluginUrls['dataexplorer']
+    if (!dataExplorerUrl) {
+      // New data-explorer is used.
+      dataExplorerUrl = '@molgenis-experimental/data-explorer/dist/index.html'
```

Comment:
what is i cant version x.y.z could we inject the url instead of declaring it here ?

Reply:
Kan, maar dan moet dat via de backend.